embedchain/loaders/audio.py

```diff
@@ -1,6 +1,8 @@
-import os
```

Comment:
@PrashantDixit0 Why did you had to change this file or is it because of lint?

Reply:
May be It changed while pulling updated code. I'll add it again no issues in that